client/src/legacy/LeftAndMain.js

```diff
@@ -959,78 +962,66 @@ $.entwine('ss', function($) {
     },
 
     showLoginDialog: function() {
-      var tempid = $('body').data('member-tempid'),
-        dialog = $('.leftandmain-logindialog'),
-        url = 'CMSSecurity/login';
-
       // Force regeneration of any existing dialog
-      if(dialog.length) dialog.remove();
-
-      // Join url params
-      url = $.path.addSearchParams(url, {
-        'tempid': tempid,
-        'BackURL': window.location.href
-      });
+      let dialog = $('.leftandmain__login-dialog');
+      if (dialog.length) {
+        dialog.destroy();
+      }
 
-      // Show a placeholder for instant feedback. Will be replaced with actual
-      // form dialog once its loaded.
-      dialog = $('<div class="leftandmain-logindialog"></div>');
-      dialog.attr('id', new Date().getTime());
-      dialog.data('url', url);
+      // Create container
+      dialog = $('<div class="leftandmain__login-dialog" />');
       $('body').append(dialog);
-    }
+      dialog.open();
+    },
   });
 
-  // Login dialog page
-  $('.leftandmain-logindialog').entwine({
-    onmatch: function() {
-      this._super();
-
-      // Create jQuery dialog
-      this.ssdialog({
-        iframeUrl: this.data('url'),
-        dialogClass: "leftandmain-logindialog-dialog",
-        autoOpen: true,
-        minWidth: 500,
-        maxWidth: 500,
-        minHeight: 370,
-        maxHeight: 400,
-        closeOnEscape: false,
-        open: function() {
-          $('.ui-widget-overlay').addClass('leftandmain-logindialog-overlay');
-        },
-        close: function() {
-          $('.ui-widget-overlay').removeClass('leftandmain-logindialog-overlay');
-        }
-      });
-    },
-    onunmatch: function() {
-      this._super();
+  /**
+   * Login dialog page
+   * Selector must match string in CMSSecurity_success.ss message callback
+  */
+  $('.leftandmain__login-dialog').entwine({
+    destroy() {
+      this.close();
+      this.remove();
     },
-    open: function() {
-      this.ssdialog('open');
+    close() {
+      this.renderModal(false);
     },
-    close: function() {
-      this.ssdialog('close');
+    open() {
+      this.renderModal(true);
     },
-    toggle: function(bool) {
-      if(this.is(':visible')) this.close();
-      else this.open();
+
+    renderModal(show) {
+      // Build properties
+      const tempid = $('body').data('member-tempid');
+      const url = $.path.addSearchParams('CMSSecurity/login', {
+        tempid,
+        BackURL: window.location.href,
+      });
+
+      // todo add URL to iframe
+      ReactDOM.render(
+        <LoginDialog className="login-dialog" bodyClassName="login-dialog__body"
```

Comment:
could we have these classes added by the component? and concatenate any other respective classNames.

Also, please apply each attribute to a new line for readability purposes